File: app/src/main/java/com/example/msimple/data/vos/ProgramVO.java
package com.example.msimple.data.vos;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ProgramVO {




        @SerializedName("program-id")
        private String programId;

        @SerializedName("title")
        private String title;

        @SerializedName("image")
        private String image;

        @SerializedName("average-lengths")
        private List<Integer> averageLengths;

        @SerializedName("description")
        private String description;

        @SerializedName("sessions")
        private List<SessionVO> sessionsList;



        public String getProgramId() {
            return programId;
        }

        public String getTitle() {
            return title;
        }

        public String getImage() {
            return image;
        }

        public List<Integer> getAverageLengths() {
            return averageLengths;
        }

        public String getDescription() {
            return description;
        }

        public List<SessionVO> getSessionsList() {
            return sessionsList;
        }

}

File: app/src/main/java/com/example/msimple/data/vos/ProgramVO.kt
package com.example.msimple.data.vos

import com.google.gson.annotations.SerializedName

class ProgramVO {


    @SerializedName("program-id")
    val programId: String? = null

    @SerializedName("title")
    val title: String? = null

    @SerializedName("image")
    val image: String? = null

    @SerializedName("average-lengths")
    val averageLengths: List<Int>? = null

    @SerializedName("description")
    val description: String? = null

    @SerializedName("sessions")
    val sessionsList: List<SessionVO>? = null

}
